test/integration/find_spec.mjs

```diff
@@ -39,35 +39,49 @@ describe("find bar", () => {
     it("must highlight text in the right position", async () => {
       await Promise.all(
         pages.map(async ([browserName, page]) => {
+          // Highlight all occurrences of the letter A (case insensitive).
           await page.click("#viewFind");
           await page.waitForSelector("#viewFind", { hidden: false });
           await page.type("#findInput", "a");
           await page.click("#findHighlightAll");
           await page.waitForSelector(".textLayer .highlight");
-          // The PDF has the text "AB BA" in a monospace font.
-          // Make sure we have the right number of highlighted divs.
+
+          // The PDF file contains the text 'AB BA' in a monospace font on a
+          // single line. Check if the two occurrences of A are highlighted.
           const highlights = await page.$$(".textLayer .highlight");
           expect(highlights.length).withContext(`In ${browserName}`).toEqual(2);
-          const glyphWidth = 15.98; // From the PDF.
-          const pageDiv = await page.$(".page canvas");
-          const pageBox = await pageDiv.boundingBox();
+
+          // Calculate the dimensions of the highlights. The font data in the
+          // PDF describes the size of the glyphs (and therefore the size of the
+          // highlights), but the viewer applies extra padding to them. For the
+          // comparison we therefore subtract the padding, which we calculate
+          // using the unpadded parent element, from the highlight's height.
+          const parentSpan = (await highlights[0].$$("xpath/.."))[0];
+          const parentBox = await parentSpan.boundingBox();
           const firstA = await highlights[0].boundingBox();
           const secondA = await highlights[1].boundingBox();
-          // Subtract the page offset from the text bounding boxes;
-          firstA.x -= pageBox.x;
-          firstA.y -= pageBox.y;
-          secondA.x -= pageBox.x;
-          secondA.y -= pageBox.y;
-          // They should be on the same line.
+          firstA.height -= firstA.height - parentBox.height;
```

Comment:
Isn't it equivalent to `firstA.height = parentBox.height` ?

Reply:
Yes, indeed. I have amended the patch to simplify this expression. Thanks!